mongodbatlas/data_source_mongodbatlas_search_analyzers.go

```diff
@@ -0,0 +1,115 @@
+package mongodbatlas
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	matlas "go.mongodb.org/atlas/mongodbatlas"
+	"net/http"
+)
+
+func dataSourceMongoDBAtlasSearchAnalyzers() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceMongoDBAtlasSearchAnalyzersRead,
+		Schema: map[string]*schema.Schema{
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"cluster_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"page_num": {
+				Type:     schema.TypeInt,
+				Optional: true,
+			},
+			"items_per_page": {
+				Type:     schema.TypeInt,
+				Optional: true,
+			},
+			"results": {
+				Type:     schema.TypeSet,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"base_analyzer": {
+							Type:     schema.TypeString,
+							Computed: true,
+							Required: false,
+						},
+						"ignore_case": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"max_token_length": {
+							Type:     schema.TypeInt,
+							Optional: true,
+						},
+						"name": {
+							Type:     schema.TypeString,
+							Optional: false,
+						},
+						"stem_exclusion_set": {
+							Type:     schema.TypeSet,
+							Optional: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+						"stopwords": {
+							Type:     schema.TypeSet,
+							Optional: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+					},
+				},
+			},
+			"total_count": {
```

Comment:
Is this one necessary?

Reply:
I took the projects datasource as example for this, I'm not sure if this is really necessary for this feature